wifiphisher/data/phishing-pages/wifi_connect/config.ini

```diff
@@ -3,3 +3,4 @@ Name: Network Manager Connect
 Description: Imitates the behavior of the network manager. This template shows Chrome's "Connection Failed" page and displays a network manager window through the page asking for the pre-shared key. Currently, the network managers of Windows and MAC OS are supported.
 
 [context]
+enterprise:0
```

Comment:
You need to describe in a comment the purpose of this flag. How will the users know what it is?